src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -357,7 +357,8 @@ export default class Pilot {
         }
 
         const airport = AirportController.airport_get();
-        const descentAltitude = Math.min(aircraftModel.altitude, this._mcp.altitude);
+        const currentAltitude = _floor(aircraftModel.altitude, -2);
```

Comment:
Why floor here instead of `Math.min`?

Reply:
The Math.min part is still there:  `const descentAltitude = Math.min(currentAltitude, this._mcp.altitude);`
The aircraft's altitude is just floored before comparing it with mcp.altitude.